diff --git a/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Worksheet/ColumnCellIterator.php b/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Worksheet/ColumnCellIterator.php
index 00dc6ac8d03..a319684ccf1 100644
--- a/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Worksheet/ColumnCellIterator.php
+++ b/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Worksheet/ColumnCellIterator.php
@@ -111,6 +111,7 @@ class ColumnCellIterator extends CellIterator
     /**
      * Rewind the iterator to the starting row.
      */
+    #[\ReturnTypeWillChange]
     public function rewind()
     {
         $this->currentRow = $this->startRow;
@@ -121,6 +122,7 @@ class ColumnCellIterator extends CellIterator
      *
      * @return null|\PhpOffice\PhpSpreadsheet\Cell\Cell
      */
+    #[\ReturnTypeWillChange]
     public function current()
     {
         return $this->worksheet->getCellByColumnAndRow($this->columnIndex, $this->currentRow);
@@ -131,6 +133,7 @@ class ColumnCellIterator extends CellIterator
      *
      * @return int
      */
+    #[\ReturnTypeWillChange]
     public function key()
     {
         return $this->currentRow;
@@ -139,6 +142,7 @@ class ColumnCellIterator extends CellIterator
     /**
      * Set the iterator to its next value.
      */
+    #[\ReturnTypeWillChange]
     public function next()
     {
         do {
@@ -165,6 +169,7 @@ class ColumnCellIterator extends CellIterator
      *
      * @return bool
      */
+    #[\ReturnTypeWillChange]
     public function valid()
     {
         return $this->currentRow <= $this->endRow && $this->currentRow >= $this->startRow;
diff --git a/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Worksheet/ColumnIterator.php b/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Worksheet/ColumnIterator.php
index 80b548d6197..c449bad2506 100644
--- a/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Worksheet/ColumnIterator.php
+++ b/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Worksheet/ColumnIterator.php
@@ -122,6 +122,7 @@ class ColumnIterator implements \Iterator
     /**
      * Rewind the iterator to the starting column.
      */
+    #[\ReturnTypeWillChange]
     public function rewind()
     {
         $this->currentColumnIndex = $this->startColumnIndex;
@@ -132,6 +133,7 @@ class ColumnIterator implements \Iterator
      *
      * @return Column
      */
+    #[\ReturnTypeWillChange]
     public function current()
     {
         return new Column($this->worksheet, Coordinate::stringFromColumnIndex($this->currentColumnIndex));
@@ -142,6 +144,7 @@ class ColumnIterator implements \Iterator
      *
      * @return string
      */
+    #[\ReturnTypeWillChange]
     public function key()
     {
         return Coordinate::stringFromColumnIndex($this->currentColumnIndex);
@@ -150,6 +153,7 @@ class ColumnIterator implements \Iterator
     /**
      * Set the iterator to its next value.
      */
+    #[\ReturnTypeWillChange]
     public function next()
     {
         ++$this->currentColumnIndex;
@@ -168,6 +172,7 @@ class ColumnIterator implements \Iterator
      *
      * @return bool
      */
+    #[\ReturnTypeWillChange]
     public function valid()
     {
         return $this->currentColumnIndex <= $this->endColumnIndex && $this->currentColumnIndex >= $this->startColumnIndex;
diff --git a/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Worksheet/Iterator.php b/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Worksheet/Iterator.php
index 348ca96f400..075a701cc69 100644
--- a/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Worksheet/Iterator.php
+++ b/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Worksheet/Iterator.php
@@ -42,6 +42,7 @@ class Iterator implements \Iterator
     /**
      * Rewind iterator.
      */
+    #[\ReturnTypeWillChange]
     public function rewind()
     {
         $this->position = 0;
@@ -52,6 +53,7 @@ class Iterator implements \Iterator
      *
      * @return Worksheet
      */
+    #[\ReturnTypeWillChange]
     public function current()
     {
         return $this->subject->getSheet($this->position);
@@ -62,6 +64,7 @@ class Iterator implements \Iterator
      *
      * @return int
      */
+    #[\ReturnTypeWillChange]
     public function key()
     {
         return $this->position;
@@ -70,6 +73,7 @@ class Iterator implements \Iterator
     /**
      * Next value.
      */
+    #[\ReturnTypeWillChange]
     public function next()
     {
         ++$this->position;
@@ -80,6 +84,7 @@ class Iterator implements \Iterator
      *
      * @return bool
      */
+    #[\ReturnTypeWillChange]
     public function valid()
     {
         return $this->position < $this->subject->getSheetCount() && $this->position >= 0;
diff --git a/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Worksheet/RowCellIterator.php b/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Worksheet/RowCellIterator.php
index 3dce799c284..8094d754a62 100644
--- a/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Worksheet/RowCellIterator.php
+++ b/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Worksheet/RowCellIterator.php
@@ -113,6 +113,7 @@ class RowCellIterator extends CellIterator
     /**
      * Rewind the iterator to the starting column.
      */
+    #[\ReturnTypeWillChange]
     public function rewind()
     {
         $this->currentColumnIndex = $this->startColumnIndex;
@@ -123,6 +124,7 @@ class RowCellIterator extends CellIterator
      *
      * @return \PhpOffice\PhpSpreadsheet\Cell\Cell
      */
+    #[\ReturnTypeWillChange]
     public function current()
     {
         return $this->worksheet->getCellByColumnAndRow($this->currentColumnIndex, $this->rowIndex);
@@ -133,6 +135,7 @@ class RowCellIterator extends CellIterator
      *
      * @return string
      */
+    #[\ReturnTypeWillChange]
     public function key()
     {
         return Coordinate::stringFromColumnIndex($this->currentColumnIndex);
@@ -141,6 +144,7 @@ class RowCellIterator extends CellIterator
     /**
      * Set the iterator to its next value.
      */
+    #[\ReturnTypeWillChange]
     public function next()
     {
         do {
@@ -165,6 +169,7 @@ class RowCellIterator extends CellIterator
      *
      * @return bool
      */
+    #[\ReturnTypeWillChange]
     public function valid()
     {
         return $this->currentColumnIndex <= $this->endColumnIndex && $this->currentColumnIndex >= $this->startColumnIndex;
diff --git a/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Worksheet/RowIterator.php b/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Worksheet/RowIterator.php
index 009ff5de430..8b83ea4ec42 100644
--- a/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Worksheet/RowIterator.php
+++ b/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Worksheet/RowIterator.php
@@ -117,6 +117,7 @@ class RowIterator implements \Iterator
     /**
      * Rewind the iterator to the starting row.
      */
+    #[\ReturnTypeWillChange]
     public function rewind()
     {
         $this->position = $this->startRow;
@@ -127,6 +128,7 @@ class RowIterator implements \Iterator
      *
      * @return Row
      */
+    #[\ReturnTypeWillChange]
     public function current()
     {
         return new Row($this->subject, $this->position);
@@ -137,6 +139,7 @@ class RowIterator implements \Iterator
      *
      * @return int
      */
+    #[\ReturnTypeWillChange]
     public function key()
     {
         return $this->position;
@@ -145,6 +148,7 @@ class RowIterator implements \Iterator
     /**
      * Set the iterator to its next value.
      */
+    #[\ReturnTypeWillChange]
     public function next()
     {
         ++$this->position;
@@ -163,6 +167,7 @@ class RowIterator implements \Iterator
      *
      * @return bool
      */
+    #[\ReturnTypeWillChange]
     public function valid()
     {
         return $this->position <= $this->endRow && $this->position >= $this->startRow;
